src/components/KPagination/PaginationOffset.vue

```diff
@@ -1,118 +1,57 @@
 <template>
   <div class="pagination-offset-button-container">
     <KButton
-      appearance="secondary"
+      appearance="tertiary"
       aria-label="Go to the previous page"
       class="pagination-button"
-      :class="{ disabled: prevButtonDisabled }"
-      data-testid="prev-btn"
-      :disabled="prevButtonDisabled"
-      @click.prevent="getPrevOffset"
+      data-testid="previous-button"
+      :disabled="previousButtonDisabled"
+      type="button"
+      @click.prevent="emit('getPreviousOffset')"
     >
       <template #icon>
-        <KIcon
```

Comment:
Aren't we deprecating the icon slot in favor of using KButton's default slot?

Same question for all instances

Reply:
We are in beta. Right now to make the button square when only using icon you need to use `icon` slot. After we've deprecated the KIcon, the `icon` prop in KButton will handle that (it will become boolean vs. now it's a string that's passed down to underlying KIcon).

Reply:
We can't - it won't be square.

Reply:
...we are everywhere else 😬

Reply:
Slot is what we instruct to use when using only icon in KButton to make it square: https://alpha--kongponents.netlify.app/components/button.html#icon-1

Not using the slot results in KButton having more padding on left and right (compare the screenshots).

![Screenshot 2024-02-29 at 12 51 45 PM](https://github.com/Kong/kongponents/assets/36751160/e7d109b5-6cad-4579-af88-a474a7806d2d)
![Screenshot 2024-02-29 at 12 51 10 PM](https://github.com/Kong/kongponents/assets/36751160/aa4f2285-891f-480a-917d-5efea2e02d1e)